docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+TaskPlugin=task/none             # Task plugin
+#
+# Node definitions
+NodeName=compute[1-16] CPUs=1 State=UNKNOWN
+#
+# Partition definitions
+PartitionName=test Nodes=compute[1-16] Default=YES MaxTime=INFINITE State=UP
+:::
+
+Please refer to the [Slurm documentation](https://slurm.schedmd.com/documentation.html) for a complete list of available parameters and their meanings.
+
+(using-sbatch)=
+## Job Submission and Monitoring in Slurm
+To submit your job script to Slurm, you use the `sbatch` command:
+
+:::{code} bash
+sbatch job_script.sh
+:::
+
+Slurm will then schedule your job to run when resources are available. It returns a job ID that you can use to monitor your job's status.
+
+To check the status of your job, you use the `squeue` command:
+
+:::{code} bash
+squeue -u *<username>*
+:::
+
+To cancel a job, you use the `scancel` command with the job ID:
+
+:::{code} bash
+scancel *<job_id>*
+:::
+
+To check detailed information about a job, use the `scontrol` command:
+:::{code} bash
+scontrol show job *<job_id>*
+:::
+
+This information is crucial for managing your jobs and ensuring they are running as expected.
+
+## Batch Jobs: `sbatch` Command
+The `sbatch` command is used to submit a job script for later execution. The script includes the `SBATCH` directives that control the job parameters like the number of nodes, CPUs per task, job name, etc.
+
+### Syntax: `sbatch`
+:::{code} bash
+sbatch [options] *<script_file>*
+:::
+
+### Options and Usage: `sbatch`
+- `n, --ntasks=*<number>*` : specify the number of tasks
+- `N, --nodes=<minnodes[-maxnodes]>` : specify the number of nodes
+- `J, --job-name=<jobname>` : specify a name for the job
+
+### Code Example of Job Submission Script
+:::{code} bash
+#!/bin/bash
+#SBATCH -J MyJob               # Job name
+#SBATCH -N 2                   # Number of nodes
+#SBATCH -n 16                  # Number of tasks
+#SBATCH -o output_%j.txt       # Standard output file
+#SBATCH -e error_%j.txt        # Standard error file
+
+# Your program/command here
+srun ./my_program
+:::
+
+To submit this job script, save it as `my_job.sh` and run:
+:::{code} bash
+sbatch my_job.sh
+:::
+
+### Examples using sbatch
+**Job request: one node**
+Run a job on one node for four hours on the short partition:
+
+:::{code} bash
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --time=4:00:00
+#SBATCH --job-name=MyJobName
+# SBATCH --partition=short
+
+# <commands to execute>
+:::
+
+**Job request: one node with additional memory**
+The default memory per allocated core is 1GB. If calculations attempt to access more memory than allocated, Slurm automatically terminates the job. Request a specific amount of memory in the job script if calculations require more than the default. The example script below requests 100GB of memory (`--mem=100G`). Use one capital letter to abbreviate the unit of memory (`K`, `M`, `G`, `T`) with the `--mem=` option, as that is what Slurm expects to see:
```

Comment:
I think it is 2 GB (or actually 1.95 GB/core).

Reply:
Good catch! Updated default value and specified units in English.